spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -145,19 +147,21 @@ public ResponseEntity<String> manifest(@PathVariable("name") String name,
 	@RequestMapping(path = "/history/{name}", method = RequestMethod.GET)
 	@ResponseStatus(HttpStatus.OK)
 	public Collection<Release> history(@PathVariable("name") String releaseName) {
-		ArgumentSanitizer sanitizer = new ArgumentSanitizer();
 		return this.streamService.history(releaseName)
 			.stream()
-			.map(release -> {
-				if (release.getConfigValues() != null && StringUtils.hasText(release.getConfigValues().getRaw())) {
-					try {
-						release.getConfigValues().setRaw(sanitizer.sanitizeJsonOrYamlString(release.getConfigValues().getRaw()));
-					} catch (JsonProcessingException e) {
-						logger.error("history:exception:" + e, e);
-					}
-				}
-				return release;
-			}).collect(Collectors.toList());
+			.map(this::sanitizeRelease)
+			.collect(Collectors.toList());
+	}
+
+	private Release sanitizeRelease(Release release) {
+		if (release.getConfigValues() != null && StringUtils.hasText(release.getConfigValues().getRaw())) {
+			try {
+				release.getConfigValues().setRaw(sanitizer.sanitizeJsonOrYamlString(release.getConfigValues().getRaw()));
+			} catch (JsonProcessingException e) {
+				logger.warn("sanitizeRelease:exception:" + e, e);
```

Comment:
What is the `token1:token2` prefix here? My original review comment was looking for some context of the exception.